app/frontend/src/components/Navigation/NavButton.tsx

```diff
@@ -22,17 +18,38 @@ const useStyles = makeStyles((theme) => ({
   selected: {
     color: theme.palette.secondary.dark,
   },
+  label: {
+    alignSelf: "center",
+    marginTop: 0,
+  },
 }));
 
+interface NavButtonProps {
+  route: string;
+  label: string;
+  labelVariant:
```

Comment:
Can this variant type be imported from mui?

Reply:
I couldn't find it :(

Reply:
What is the type of the `variant` prop of `Typography`?

Reply:
Dunno, I just put what I saw on the [docs](https://material-ui.com/api/typography/#props)

Reply:
`Exclude<TypographyProps['variant'], undefined>` should do it. The utility type is needed because `variant` is optional on `TypographyProps` and we probably don't want `undefined` to be a possible value here.

Reply:
How did you find out what the type is?

Reply:
VS Code shows you if you write out something like `type Variant = TypographyProps['variant']` and hover over where it says `TypographyProps['variant']`. Or you can click on `TypographyProps` and it should jump you into that file with the type definition